tests/dnszone/test_dnszone.yml

```diff
@@ -14,6 +14,7 @@
   - name: Ensure zone is present.
     ipadnszone:
       ipaadmin_password: SomeADMINpassword
+      ipaapi_ldap_cache: no
```

Comment:
Why is ldap_cache disabled for this task?

Reply:
Because by default, `ldap_cache` is disabled, and I wanted to test it changing the default behavior.